app/components/floating_text_input_label/styles.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import {BORDER_DEFAULT_WIDTH, ADORNMENT_TO_INPUT_SPACE, INPUT_CONTAINER_HORIZONTAL_SPACING, INPUT_CONTAINER_VERTICAL_SPACING} from './constants';
+
+export const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
```

Comment:
I get the reason why you did this, but is breaking our code style where this styles should be defined above the component.